resources/lint/profiles/slas.raml

```diff
@@ -177,23 +164,29 @@ validations:
     targetClass: apiContract.EndPoint
     propertyConstraints:
       apiContract.path:
-        pattern: ^((\/[a-z\.][a-z0-9-]*[a-z0-9]+)|(\/{[^}]+}))+$
+        pattern: ^((\/[a-z][a-z0-9-]*[a-z0-9]+)|(\/\{[^}]+}))+$
   
   unique-display-name:
     message: displayName must be unique within an API
     targetClass: apiContract.Operation
+    # TODO: Convert this to not use functionConstraint
     functionConstraint:
       code: |
```

Comment:
Is code still supported in 4.3?

Reply:
The [release notes for 4.2](https://github.com/aml-org/amf/releases/tag/4.2.0) say it's no longer supported, but the functionality still exists. I guess it's deprecated?

I decided to leave this one for later because I'm leaving the team and because it requires input from AMF, but the person who was helping is now on vacation. I've added a story so that someone can follow up later.